dev/io.openliberty.org.apache.myfaces.4.1/build.gradle

```diff
@@ -19,7 +16,7 @@ configurations {
 }
 
 dependencies {
-   sourceJar 'org.apache.myfaces.core:myfaces-impl:4.1.0:sources'
+//    sourceJar 'org.apache.myfaces.core:myfaces-impl:4.1.0:sources'
```

Comment:
Can you open an issue to get these sources uploaded or how are you handling it?

Reply:
https://github.com/OpenLiberty/open-liberty/issues/31096

Reply:
I will also message Chuck now :)